src/components/UtilizationBar/UtilizationBar.js

```diff
@@ -0,0 +1,94 @@
+import { ProgressBar } from '../ProgressBar';
+import PropTypes from 'prop-types';
+import { Tooltip } from '../Tooltip';
+import { OverlayTrigger } from '../OverlayTrigger';
+import React from 'react';
+
+const availableTooltip = props => {
+  return (
+    <Tooltip id="availableTooltip">Available {props.max - props.now} %</Tooltip>
```

Comment:
Using `id` here is very risky, it mean I can't use this component twice.
You can generate a unique id or ask the consumer to pass it and be responsible for using different id each time using the component.

Same in `usedTooltip`.

Reply:
@sharvit Soo... `Tooltip` explicitly asks for `id`. And I have two possibilities that are not nice
- render random `id` (right now)
- override `Tooltip` from `react-bootstrap` to have `id` optional (@priley86 didn't like similar change before)